main.py

```diff
@@ -161,6 +161,10 @@ def configure_confluent_kafka_consumer(event, args):
         "path.format": args.path_format,
         "locale": args.locale,
         "timezone": args.timezone,
+        "errors.tolerance": "all",
```

Comment:
these all look right, but they should all me parameters passed through the arg parser